objectcomputing/check-ins
# Feature 2148/replace nivo with recharts radar chart teamskills

To test:
- Log in as an admin
- Go to REPORTS > Team Skills
- Select some members who have skills
- Select some skills, then run search
-  Radar chart should display, and colors should not change unless you refresh the page
- *Note:* The radar chart seems to work with only 1 skill, but it is not easy to read without the tooltip

This finishes up the work started by @Richardson-e. Adjustments:
- Removed nivo dependency
- Fixed how colors are generated for the radar chart
- Radar chart now displays tooltip with info about the distribution

There are probably additional enhancements we could make for generating the radar chart colors:
- Make sure all generated colors are distinct in hue
- Make sure all generated colors are legible (e.g. yellow, light pastels)
- Allow the user to regenerate all colors or a color of their choosing

Closes #2184
